pyro/contrib/epidemiology/compartmental.py

```diff
@@ -660,6 +688,7 @@ def heuristic(self, num_particles=1024, ess_threshold=0.5, retries=10):
                 continue
 
         # Select the most probable hypothesis.
+        # Note this ignores the .finalize() likelihood.
```

Comment:
To clarify, is `finalize` ignored during initialization? If so, mention in the `finalize` docstring?

Reply:
done